File: src/main/java/wushu/entity/Athlete.java
package wushu.entity;

import jakarta.persistence.*;
import lombok.Data;

import java.util.Date;

@Data
@Entity
public class Athlete {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String firstName;
    private String lastName;
    @Temporal(TemporalType.DATE) // Вказуємо, що тип даних - DATE
    private Date birthDate;

    @Enumerated(EnumType.STRING)
    private ProgramType programType;

    private Long userId;
}

File: frontend/src/models/athlete.ts
import {ProgramType} from "@/models/programtypes";
import {Gender} from "@/models/gender";

export interface AthleteDTO{
    id?: number | null;
    firstName: string;
    lastName: string;
    birthDate: string;
    gender: Gender
    programType: ProgramType;
    userId: number;
}